integration-tests/pom.xml

```diff
@@ -22,6 +22,9 @@
         <quarkus.build.skip>${skipTests}</quarkus.build.skip>
         <native.surefire.skip>${skipTests}</native.surefire.skip>
         <enforce-test-deps-scope.skip>true</enforce-test-deps-scope.skip>
+
+        <quarkus.config-tracking.enabled>true</quarkus.config-tracking.enabled>
+        <quarkus.native.container-build>true</quarkus.native.container-build>
```

Comment:
Can you clarify why you added `<quarkus.native.container-build>true</quarkus.native.container-build>`?

Reply:
This is required by the [Quarkus caching extension](https://github.com/gradle/develocity-build-config-samples/tree/main/quarkus-build-caching-extension#limitations) to ensure native builds are as reproducible as possible and make caching consistent.